From 307045f66690cd8c454ce3cb8259a31b18cc48b0 Mon Sep 17 00:00:00 2001
From: Natalia Baliakina <natalia.baliakina@fxbackoffice.com>
Date: Fri, 24 Oct 2025 15:31:00 +0300
Subject: [PATCH 2/2] feat: Demonstrate Logic Hot Reload

---
 Pages/Index.cshtml.cs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Pages/Index.cshtml.cs b/Pages/Index.cshtml.cs
index 3605180..07a38ef 100644
--- a/Pages/Index.cshtml.cs
+++ b/Pages/Index.cshtml.cs
@@ -14,6 +14,6 @@ public class IndexModel : PageModel
 
     public void OnGet()
     {
-        Message = "This is the initial logic.";
+        Message = "The code logic has been updated by Hot Reload!";
     }
 }
\ No newline at end of file
-- 
2.50.1

